tst/EnergyPlus/unit/AirflowNetworkHVAC.unit.cc

```diff
@@ -19766,6 +19827,12 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_ZoneOrderTest)
     state->afn->AirflowNetworkNodeData(2).EPlusNodeNum = 4;
     // Attic_Unit1
     state->afn->AirflowNetworkNodeData(3).EPlusNodeNum = 0;
+
+    // Check that the validation fails if the AFN exhaust fan is not well setup
+    int exhaustFanInletNodeIndex = state->afn->MultizoneCompExhaustFanData(1).InletNode;
+    state->afn->MultizoneCompExhaustFanData(1).InletNode = 6;
+    state->afn->ValidateExhaustFanInputOneTimeFlag = true;
+    EXPECT_THROW(state->afn->validate_exhaust_fan_input(), std::runtime_error);
```

Comment:
Mess with the exhaust fan node to make sure that the if statement that was moved still creates a fatal error.